pkg/trojan/message.go

```diff
@@ -63,27 +55,27 @@ var (
 	// ErrVarLenTargets is returned when the given target list for a trojan chunk has addresses of different lengths
 	ErrVarLenTargets = errors.New("target list cannot have targets of different length")
 
-	ErrInvalidHasher = errors.New("hasher pool not initialized")
+	// ErrUnMarshallingTrojanMessage is returned when a trojan message could not be de-serialized
+	ErrUnMarshallingTrojanMessage = errors.New("trojan message unmarshall error")
```

Comment:
just call this `ErrUnmarshal`  trojan message is implicit in the package, it is ungo to repeat it